packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -117,6 +117,18 @@ - (instancetype)initWithFrame:(CGRect)frame
     if ([initialUrl isKindOfClass:[NSString class]]) {
       [self loadUrl:initialUrl];
     }
+    
+    
+    BOOL hasPullToRefresh = [args[@"hasPullToRefresh"] boolValue];
+
+    if (hasPullToRefresh) {
+      UIRefreshControl *refreshControl = [[UIRefreshControl alloc] init];
+      [refreshControl setTintColor:[UIColor grayColor]];
+      [refreshControl addTarget:self action:@selector(refreshWebview) forControlEvents:UIControlEventValueChanged];
+      if (@available(iOS 10.0, *)) {
```

Comment:
Why is this check after creating the refresh control, given that it will never be used when this is false?